chisel/client.go

```diff
@@ -48,10 +48,7 @@ func (client *Client) CreateTunnel(tunnelConfig agent.TunnelConfig) error {
 
 	client.chiselClient = chiselClient
 
-	ctx, cancel := context.WithTimeout(context.Background(), tunnelClientTimeout)
-	defer cancel()
-
-	err = chiselClient.Start(ctx)
+	err = chiselClient.Start(context.Background())
```

Comment:
I wonder if there is any impact to that change as the context has no timeout anymore?

Reply:
That timeout was not working properly, and I had to remove it because with the new version of Chisel the context cancellation was halting all the communications.